### DISCUSSION TITLE
Logger Events

### DISCUSSION THREAD

**Discussion by sasha172017** [adonisjs/core#2755] [Help]
I need to send somesing error, fatal, warn etc in telegram. Does adonis has hooks, intreceptor or event that i`ll can send message error and (its very important) stack trace using Logger? Or is it missing? thank you)

**Comment by RomainLanz** [comment]
Hey @sasha172017! 👋 

You can use the Exception Handler to report your exception using the service you like.
📚 https://docs.adonisjs.com/guides/exception-handling#error-reporting

**Reply by sasha172017** [reply]
Thank you, but if it`s possibly maybe you will can add event, hook, trap, interceptor to Logger)
